utils/reviewcomment.go

```diff
@@ -26,15 +26,74 @@ const (
 	SastComment       ReviewCommentType = "Sast"
 )
 
-func AddReviewComments(repo *Repository, pullRequestID int, client vcsclient.VcsClient, issues *IssuesCollection) (err error) {
-	if err = deleteOldReviewComments(repo, pullRequestID, client); err != nil {
-		err = errors.New("couldn't delete pull request review comment: " + err.Error())
+func HandlePullRequestCommentsAfterScan(issues *IssuesCollection, repo *Repository, client vcsclient.VcsClient, pullRequestID int) (err error) {
+	if !repo.Params.KeepPreviousComments {
+		log.Debug("Looking for an existing Frogbot pull request comments. Deleting them if exists...")
```

Comment:
...request comments. Deleting them if exists... --> ...request comment. Deleting it if it exists...